deps/ccommon/rust/cc_binding/build.rs

```diff
@@ -3,57 +3,64 @@ extern crate bindgen;
 extern crate failure;
 
 use std::env;
+use std::ffi::OsString;
 use std::fs;
 use std::io;
-use std::io::BufReader;
 use std::io::prelude::*;
-use std::ffi::OsString;
+use std::io::BufReader;
 use std::os::unix::fs as unix_fs;
 use std::path::Path;
```

Comment:
I know it was already like this... but can you change the two use declarations for `std::path` to a single use?